lib/src/app/player.js

```diff
@@ -231,6 +235,17 @@ class CastPlayer {
     return this.OOPlayer ? this.OOPlayer.getDuration() : null;
   }
 
+  /**
+   * Turns on/off closed captions
+   */
+  toggleClosedCaptions () {
+    if (!this.OOPlayer) {
+      logger.warn('Attempt to toggle closed captions for player that is not loaded yet');
+      return;
+    }
+    this.OOPlayer.toggleClosedCaptions();
```

Comment:
Shouldn't this be unit tested?

Reply:
I couldn't find any tests in the receiver app. Which test runner\mocking lib should I use here?

Reply:
I vote for jest